public/components/ui_action_context_menu.tsx

```diff
@@ -31,6 +31,7 @@ export const ActionContextMenu = (props: Props) => {
           trigger: AI_ASSISTANT_QUERY_EDITOR_TRIGGER as any,
         })),
         closeMenu: () => setOpen(false),
+        title: '',
```

Comment:
does this empty title is needed here?

Reply:
Yes, the default title is `Options` but we don't need a title, so set it to `''`